docs/development/i18n/index.md

```diff
@@ -9,18 +9,18 @@ react-i18next libraries.
 
 ## Default language, and locales
 
-We started with an international English, and that will be the fallback language.
+We started with an international English and that will be the fallback language.
 
-Now we're starting with locales familiar, and will accept
-translations through github.
+Now we're starting with locales familiar and will accept
```

Comment:
I guess this should be:
"Now we're starting with locales familiar **_to us_** and will accept"